README.md

```diff
@@ -24,15 +24,21 @@ Follow the [Prerequisites](https://github.com/esp-rs/esp-idf-template#prerequisi
 The examples could be built and flashed conveniently with [`cargo-espflash`](https://github.com/esp-rs/espflash/). To run e.g. `wifi` on an e.g. ESP32-C3:
 (Swap the Rust target and example name with the target corresponding for your ESP32 MCU and with the example you would like to build)
 
-with `cargo-espflash` V2+:
+with `cargo-espflash`:
 ```sh
-$ cargo espflash flash --target riscv32imc-esp-espidf --example wifi --monitor
+$ MCU=esp32c3 cargo espflash flash --target riscv32imc-esp-espidf --example wifi --monitor
 ```
 
-with older `cargo-espflash`:
-```sh
-$ cargo espflash --target riscv32imc-esp-espidf --example wifi --monitor /dev/ttyUSB0
-```
+| MCU | "--target" |
+| --- | ------ |
+| esp32c2 | riscv32imc-esp-espidf |
+| esp32c3| riscv32imc-esp-espidf |
+| esp32c6| riscv32imac-esp-espidf |
+| esp32h2 | riscv32imac-esp-espidf |
+| esp32 | xtensa-esp32-espidf |
+| esp32s2 | xtensa-esp32s2-espidf |
+| esp32s3 | xtemsa-esp32s3-espidf |
```

Comment:
Typo: `xtemsa` instead of `xtensa`.

Reply:
Could you provide mappings for esp32c5 (I know it's not out yet, but support of it is ready) as well as for esp32p4, which already has the `riscv32imafc-esp-espidf` target upstreamed?

Reply:
yeah will do

Reply:
though we don't know the target of c5. it could be anything at this point, but i will add p4 as it is on the horizion